src/schnetpack/data/loader.py

```diff
@@ -21,7 +21,9 @@ def _atoms_collate_fn(batch):
         dict[str->torch.Tensor]: mini-batch of atomistic systems
     """
     elem = batch[0]
-    idx_keys = {structure.idx_i, structure.idx_j}
+    idx_keys = {structure.idx_i, structure.idx_j, structure.idx_i_triples}
```

Comment:
i think the idx_j_triples are now treated a second time in ll 28-29, right?

Reply:
Fixed.